.gitignore

```diff
@@ -12,6 +12,7 @@ cache-ovm
 
 l2geth/build/bin
 packages/contracts/deployments/custom
+packages/contracts/coverage*
```

Comment:
Output of test coverage is: 
1. A json file: `packages/contracts/coverage.json`
2. A directory with other stuff `packages/contracts/coverage/`